datalad/distribution/tests/test_get.py

```diff
@@ -264,6 +264,7 @@ def test_get_recurse_subdatasets(src, path):
 def test_get_install_missing_subdataset(src, path):
 
     ds = install(path=path, source=src)
+    ds.create(force=True)  # force, to cause dataset initialization
```

Comment:
Remember that we have 3 types of dataset. Source path is pure git/annex - not a dataset